File: Safari-609.1.11.1/LayoutTests/http/tests/misc/resources/check-unnamed-file-included-in-formdata.php
<?php
header("Content-Type: text/html; charset=UTF-8");
?>
<html>
<head>
<script>
    if (window.testRunner)
        testRunner.notifyDone();
</script>
</head>
<body>
<h1>This tests verifies that files without a name are included in the multipart form data:</h1>
<pre>
<?php
if (isset($_FILES['data'])) {
    print "Test passed.";
} else {
    print "Test failed.";
}
?>
</pre>
</body>
</html>

File: LayoutTests/http/tests/misc/resources/check-unnamed-file-included-in-formdata.py
#!/usr/bin/env python3

import cgi
import sys

form = cgi.FieldStorage()
data = form.getvalue('data')

sys.stdout.write(
    'Content-Type: text/html; charset=UTF-8\r\n\r\n'
    '<html>\n'
    '<head>\n'
    '<script>\n'
    '    if (window.testRunner)\n'
    '        testRunner.notifyDone();\n'
    '</script>\n'
    '</head>\n'
    '<body>\n'
    '<h1>This tests verifies that files without a name are included in the multipart form data:</h1>\n'
    '<pre>\n'
)

if data is not None:
    sys.stdout.write('Test passed.')
else:
    sys.stdout.write('Test failed.')

sys.stdout.write(
    '</pre>\n'
    '</body>\n'
    '</html>\n'
)